src/main/java/pro/belbix/ethparser/model/VaultsResponse.java

```diff
@@ -0,0 +1,61 @@
+package pro.belbix.ethparser.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;;
+
+@Data
+@NoArgsConstructor
+public class VaultsResponse {
```

Comment:
not necessary to create another response model
use RestResponse, just put model as json in the data field